.github/ISSUE_TEMPLATE/content-template.yaml

```diff
@@ -25,7 +24,4 @@ body:
         - No.
         - Maybe (please specify)
```

Comment:
@NukeManDan : can we take out the "(please specify)" ?

Reply:
why?

Reply:
Because there's nowhere for someone to specify 😆

Reply:
```suggestion
        - Maybe (please specify above)
```